src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/languageConstructions/ClassConstantCanBeUsedInspector.java

```diff
@@ -171,8 +171,8 @@ public void applyFix(@NotNull Project project, @NotNull ProblemDescriptor descri
 
                     /* check all use-statements and use imported name for QF */
                     for (PhpUseList use : PsiTreeUtil.findChildrenOfType(file, PhpUseList.class)) {
-                        /* do not process `function() use () {}` constructs */
-                        if (use.getParent() instanceof Function) {
+                        /* do not process `function() use () {}` constructs or class traits */
+                        if (use.getParent() instanceof Function || use.getParent() instanceof PhpClass) {
```

Comment:
Ah, gotcha. I just assumed that Psi would cache the element after the first call. I'll introduce the change in a bit